array.c

```diff
@@ -4143,14 +4143,16 @@ ary_recycle_hash(VALUE hash)
  *  Array Difference
  *
  *  Returns a new array that is a copy of the original array, removing any
- *  items that also appear in +other_ary+. The order is preserved from the
- *  original array.
+ *  items that also appear in +other_ary+. The order of the original
+ *  array is preserved.
```

Comment:
The previous version is clearer (the new statement might be misunderstood as refering to the original array not being modified).